accounts/views.py

```
@@ -1,3 +1,31 @@
-from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic import DetailView, UpdateView, TemplateView
```

Ruff (0.8.2)

`django.views.generic.DetailView` imported but unused

Remove unused import: `django.views.generic.DetailView`

(F401)